src/js/utils/validator.ts

```diff
@@ -47,15 +47,15 @@ export function isRtmp(file, type) {
  * @param {string} type - The type of the media parsed from a feed or the media url.
  * @returns {boolean} Is `path` a YouTube link or does `type` equal 'youtube'?
  */
-export function isYouTube(path, type) {
+export function isYouTube(path: string, type: string): boolean {
     return (type === 'youtube') || (/^(http|\/\/).*(youtube\.com|youtu\.be)\/.+/).test(path);
 }
 
 /**
  * @param {string} value - The variable to test.
  * @returns {string} The typeof object, 'array' or 'null'.
  */
-export function typeOf(value) {
+export function typeOf(value: any): string {
```

Comment:
Question about this function:
It looks like this method is only used for tests, but I was wondering about what this 'value' is? The comment implies it is a string, but it also seems to be checking for arrays as well?

Reply:
value: any

Reply:
value: any

Reply:
value: any

Reply:
value: any